jest-setup.ts

```diff
@@ -0,0 +1,6 @@
+import '@testing-library/jest-dom';
+import { matchers } from '@emotion/jest';
+
+expect.extend(matchers);
+
+export {};
```

Comment:
I figure this is boilerplate standard... but out of curiosity, what's it function?
may be useful to add a comment (e.g. within the curlies) to explain something so eager beavers like myself don't go deleting that line thinking it's cruft.

Reply:
Good idea. It's just to make TS happy so it's a valid module seen as we're not exporting anything, just setting up some stuff